lib/kwtool/accounts/schemas/user.ex

```diff
@@ -10,8 +10,9 @@ defmodule Kwtool.Accounts.Schemas.User do
     field :encrypted_password, :string
     field :password, :string, virtual: true
     field :password_confirmation, :string, virtual: true
-
```

Comment:
We should add a line break for this to separate field section and timestamp.

Reply:
updated.

Reply:
Updated! Thanks 👍